amlip_docs/rst/user_manual/nodes/async_edge.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_nodes_async_edge:
+
+######################
+Asynchronous Edge Node
+######################
+
+This node is able to send data serialized as :ref:`user_manual_datatype_inference` and it receives an Inference as :ref:`user_manual_datatype_inference_solution`.
+
+Example of Usage
+================
+
+Users can use method :code:`request_inference` to send new data.
+The thread calling this method will wait until the whole process has finished and the *Inference* has arrived from the :ref:`user_manual_nodes_async_inference` in charge of this data.
```

Comment:
Be more specific. There is no need to make reference to internal thread, explain better that the solution will arrive async (in other thread) and how to get it (by listener)